scalardb-graphql-sample/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.9"
+services:
+  cassandra:
+    image: cassandra:3.11
+    container_name: "cassandradb-1"
+    volumes:
+      - cassandra-data:/var/lib/cassandra
+    ports:
+      - "9042:9042"
+    networks:
+      - scalar-network
+
+  schema-loader:
+    image: ghcr.io/scalar-labs/scalardb-schema-loader:3.5.0
+    container_name: "scalardb-schema-loader-1"
+    volumes:
+      - ./database.properties:/database.properties
+      - ./emoney-schema.json:/emoney-schema.json
+    command: "--config /database.properties -f /emoney-schema.json --coordinator"
+    networks:
+      - scalar-network
+    profiles:
+      - tools
```

Comment:
`profiles` is a new feature available in `version: "3.9"`. By specifying this to the `schema-loader` service, `docker-compose up` without `--profile` option doesn't start it, so we can have this service as a one-shot service.
https://stackoverflow.com/a/65957695